tests/queries/0_stateless/03310_minmax_proj_view.sql

```diff
@@ -0,0 +1,10 @@
+drop table if exists test_minmax;
+drop table if exists test_minmax_v;
+
+set enable_analyzer=1;
+
+create table test_minmax (x UInt32) engine = MergeTree order by x;
+insert into test_minmax select number + 42 from numbers(1e6);
+
+create view test_minmax_v (x UInt32) as select x from test_minmax;
+explain select min(x), max(x) from test_minmax_v;
```

Comment:
The PR is named `Support for materialzie in minmax projection` but I don't see `materialize()` is used in the test

Reply:
It's hidden inside the view.
Previously the projection was not applied because of it.